source/configuration_manual/push_notification.rst

```diff
@@ -494,3 +496,91 @@ Example with event code:
        e:log_info("Mailbox notify status " .. tostring(code))
      end
    end
+
+Chronos driver [``chronos``]
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Similar to the :ref:`push_notification_ox` the Chronos backend supports sending
+notifications on MessageNew events in case the message contains a calendar
+invite.
+
+This driver was designed for use with the `OX App Suite iCalendar
+Transport-Independent Interoperability Protocol (iTIP)
+<https://documentation.open-xchange.com/7.10.6/middleware/calendar/iTip.html#rest-api>`_,
+but can be used by any endpoint that implements the same API, not just OX App
+Suite.
+
+Configuration options:
+
+================= ======== ============= ============================================================================================================
+ Name             Required Type          Description
+================= ======== ============= ============================================================================================================
+ ``url``          **YES**  :ref:`string` The HTTP end-point (URL + authentication information) to use for sending the push notification.
+                                         Contains authentication information needed for Basic Authentication (if any). Example:
+                                         ``http<s> + "://" + <login> + ":" + <password> + "@" + <host> + ":" + <port> + "/chronos/v1/itip/pushmail"``
+
+                                         For HTTPS endpoints, system CAs are trusted by default, but internal CAs might need further configuration.
+
+                                         For further details on configuring the App Suite endpoint, see:
+                                         https://documentation.open-xchange.com/7.10.6/middleware/calendar/iTip.html#configuration2
+
+ ``max_retries``  NO       :ref:`uint`   The maximum number of retries to attempt to connect to the API endpoint. (DEFAULT: ``1``)
+
+ ``timeout``      No       :ref:`time`   Time before HTTP request to OX endpoint will timeout. (DEFAULT: ``2s``)
+
+ ``msg_max_size`` No       :ref:`size`   Maximum size a message may have to be considered for push notification sending. (DEFAULT: ``500kb``)
+================= ======== ============= ============================================================================================================
+
+Example configuration:
+
+.. code-block:: none
+
+  plugin {
+    push_notification_driver = chronos:url=http://login:pass@node1.domain.tld:8009/chronos/v1/itip/pushmail max_retries=2 timeout=2500ms msg_max_size=1mb
+  }
+
+Metadata
+--------
+
+The push notifications are enabled separately for each user using METADATA.
+Normally `OX App Suite <https://www.open-xchange.com/products/ox-app-suite/>`_
+does this internally, but for e.g. testing purposes you can do this yourself:
```

Comment:
I thought there wasn't supposed to be any METADATA usage with this plugin?

Reply:
There isn't. I was copying too much blindly from the OX push notification documentation. :grimacing: